README.md

```diff
@@ -0,0 +1,187 @@
+[![Code of Conduct](https://img.shields.io/badge/code%20of%20conduct-Ansible-yellow.svg)](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html) 
+[![AWX Mailing List](https://img.shields.io/badge/mailing%20list-AWX-orange.svg)](https://groups.google.com/g/awx-project)
+[![IRC Chat - #ansible-awx](https://img.shields.io/badge/IRC-%23ansible--awx-blueviolet.svg)](https://libera.chat)
+
+An kubernetes operator that can be used along side [Ansible AWX](https://github.com/ansible/awx) to launch automation jobs and create job templates by creating resources directly in kubernetes. This operator is built with [Operator SDK](https://github.com/operator-framework/operator-sdk) and Ansible.
+
+## Table of Contents
+
+* [AWX Resource Operator](#awx-resource-operator)
+   * [Overview](#overview)
+      * [Prerequisites](#prerequisites)
+   * [Installing the Operator](#installing-the-operator)
+   * [Usage](#usage)
+      * [AnsibleJob](#ansiblejob)
+      * [JobTemplate](#jobtemplate)
+   * [Building and Testing](#building-and-testing)
+   * [Author](#author)
+   * [Code of Conduct](#code-of-conduct)
+   * [Get Involved](#get-involved)
+
+## Overview
+
+There are currently two Custom Resources provided by the awx-resource-operator:
+
+* AnsibleJob - launches a job in the AWX instance specified in the the specified k8s secret (awx host url, token)
+* JobTemplate - creates a job template in the AWX instance specified.
+
+### Prerequisites
+
+* Install the kubernetes-based cluster of your choice:
+  * [Openshift](https://docs.openshift.com/container-platform/4.11/installing/index.html)
+  * [K8s](https://kubernetes.io/docs/setup/)
+  * [CodeReady containers](https://access.redhat.com/documentation/en-us/red_hat_openshift_local/2.5)
+  * [minikube](https://minikube.sigs.k8s.io/docs/start/)
+* Deploy AWX using the [awx-operator](https://github.com/ansible/awx-operator#basic-install)
+* Create an OAuth2 token for your user in the AWX UI
+
+
+## Installing the Operator
+
+1. Clone the awx-resource-operator
+
+```
+git clone git@github.com:ansible/awx-resource-operator.git
+```
+
+2. Log in to your cluster.
+
+```
+kubectl login <cluster-url>
+```
+
+3. Run the make target
+
+```
+NAMESPACE=awx IMG=quay.io/chadams/awx-resource-operator:latest make deploy
+```
+
+This will create a deployment named `resource-operator-controller-manager` in the namespace specified, in this case the `awx` namespace. The resulting operator pod will reconcile any AnsibleJob or JotTemplate resources created.
+
+
+## Usage
+
+Now that the awx-resource-operator is installed in your cluster, you can try creating a k8s resource to manage your AWX instance.
+
+First, you need create a k8s secret with connection information for your AWX instance though. Below is example yaml that you can save to a file, edit, and apply to your cluster.
+
+### Create an AWX Access Token
+
+Create an OAuth2 token for your user in the AWX UI.
```

Comment:
@rlopez133 I added the steps to create a personal access token in it's own section here.

Reply:
Looks good